bot/exts/moderation/incidents.py

```diff
@@ -323,6 +398,19 @@ async def process_event(self, reaction: str, incident: discord.Message, member:
         else:
             log.trace("Deletion was confirmed")
 
+        log.trace("Deleting discord links webhook message.")
+        webhook_msg_ids = await self.message_link_embeds_cache.get(incident.id)
+
+        if webhook_msg_ids:
+            webhook_msg_ids = webhook_msg_ids.split(",")
+            webhook = await self.bot.fetch_webhook(Webhooks.incidents)
+
+            for x, msg in enumerate(webhook_msg_ids):
+                await webhook.delete_message(msg)
+                log.trace(f"Deleted discord links webhook message{x}/{len(webhook_msg_ids)}")
```

Comment:
I believe there is a space missing here
```suggestion
                log.trace(f"Deleted discord links webhook message {x}/{len(webhook_msg_ids)}")
```
Besides I'd rather log the message ID than the number.